variety/profile.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from variety.Util import Util
+
+DEFAULT_PROFILE_PATH = "~/.config/variety/"
```

Comment:
I just noticed that we're hardcoding these paths - a better method going forward would be to honour XDG_CONFIG_HOME. Fortunately, [GLib](https://developer.gnome.org/glib/2.40/glib-Miscellaneous-Utility-Functions.html#g-get-user-config-dir) makes this quite easy:

```
>>> from gi.repository import GLib
>>> GLib.get_user_config_dir()
'/home/james/.config'
```

Reply:
I agree, but I'd treat this as a separate followup issue.
What about `~/.local/share/applications/` ?

Reply:
```
>>> from gi.repository import GLib
>>> GLib.get_user_data_dir()
'/home/james/.local/share'
```